packages/MorphicAPIExplorer-Tests.package/MEXLinter.class/instance/brokenTestUsesDoInsteadOfCollectOrSelect.st

```diff
@@ -0,0 +1,6 @@
+test-for-intention-revealing
+brokenTestUsesDoInsteadOfCollectOrSelect
+	"Wrongly triggers when grouping collection elements by adding them to a Dictionary. Also has problems when there are multiple dos inside each other"
```

Comment:
This is actually a very important test case since the SWT-team is very harsh with do's. Please reconsider classifying it as "dont" and putting it in `excludedTests`

Reply:
But the test actually fails when there is no need for it to fail (even the SWT team says so) which would mean that it is broken or am I missing somehing?

Reply:
Interesting. But in my opinion we should reserve broken for things that actually throw errors, not false positives. Dont should be for tests that might show actual wrong code which we still want to log at least, see my other comment